src/store/region-context.tsx

```diff
@@ -84,17 +84,21 @@ export const RegionProvider = ({ children }: RegionProviderProps) => {
         const is_dev = isLocalhost() || isTestlink()
         if (website_status) {
             const { clients_country, p2p_config } = website_status
+            if (is_p2p_enabled()) {
+                setP2PAllowedCountry(true)
```

Comment:
just a suggestion that is it possible to write the conditions inside a function and then call the function here for better readability.Can you simplify it.Thank you